test/View/Helper/FormDateTimeSelectTest.php

```diff
@@ -413,4 +413,18 @@ public function testDifferentIntlFormatRendersDifferentMarkup(): void
 
         self::assertNotEquals($longMarkup, $shortMarkup);
     }
+
+    public function testRendersDatesWithZhHansHKLocaleDatePattern(): void
+    {
+        $this->helper->setLocale('zh_Hans_HK');
+        $this->helper->setDateType(IntlDateFormatter::LONG);
```

Comment:
We need to ensure this test is always going to be useful, and clear to the future maintainters, let's add a reference to what we are really testing:
```suggestion
        $this->helper->setDateType(IntlDateFormatter::LONG);
        self::assertSame('y年M月d日 z ah:mm:ss', $this->helper->getPattern());
```